locales/en-US/runtime.json

```diff
@@ -77,15 +82,52 @@
             "stopped": "Stopped",
             "missing-types": "Waiting for missing types to be registered:"
             
+        },
+        "Flow": {
```

Comment:
I realise you're following the naming scheme of the files, but let's keep it lower-case `flow` here